src/config/statsd.ts

```diff
@@ -42,17 +44,38 @@ function hrtimer() {
 	};
 }
 
-export const elapsedTimeMetrics = (
+/**
+ * Returns a middleware function which produce duration and requests count metrics
+ * @param path The value of the "path" tag. If not set then the rout path will be used
+ */
+export const elapsedTimeMetrics = (path?: string) => (
 	req: Request,
 	res: Response,
 	next: NextFunction
 ) => {
 	const elapsedTimeInMs = hrtimer();
-	const { path, method } = req;
-	const tags = { path, method };
+	const method = req.method;
+	const pathTag = path || req.route?.path || "unknown";
```

Comment:
I defaulted to "none" for the status property for webhook metrics in the event it wasn't in the response from GH/Jira. Do you think we should keep consistent across tags?

Reply:
Changed it, but not exactly in this way, we decided to capture as much of path as we can. If we use baseUrl, we already have a part of the path if request finished in any middleware in the middle which was before the final request handler. 

So we add baseUrl + '*' when we don't know the rout path, and made it '/*' when we don't know baseUrl to make it the same format